tools/releaseBuild/azureDevOps/templates/testartifacts.yml

```diff
@@ -0,0 +1,38 @@
+parameters:
+  buildName: 'linux'
+  pool: 'Hosted Ubuntu 1604'
+
+jobs:
+- job: build_testartifacts_${{ parameters.buildName }}
+  displayName: Build test artifacts ${{ parameters.buildName }}
+  condition: succeeded()
+  pool: ${{ parameters.pool }}
+  steps:
+
+  - pwsh: |
+      Import-Module ./build.psm1
+      Start-PSBootstrap
+    displayName: Bootstrap
+
+  - pwsh: |
+      Import-Module ./build.psm1
+      New-TestPackage -Destination $(System.ArtifactsDirectory)
+    displayName: Build test package
+
+  - pwsh: |
+      if (-not (Test-Path $(System.ArtifactsDirectory)/TestPackage.zip))
```

Comment:
Suggestion: Maybe some of this PowerShell block should be moved into a new function in `./tools/ci.psm1`

Reply:
This is not for CI so, this does not belong in `ci.psm1`

Reply:
It would be good if in the future we had a similar module for release builds.